src/Host.cpp

```diff
@@ -3802,3 +3802,29 @@ void Host::setupIreDriverBugfix()
         set_USE_IRE_DRIVER_BUGFIX(true);
     }
 }
+
+std::optional<QString> Host::windowType(const QString& name) const
+{
+    if (mpConsole->mLabelMap.contains(name)) {
+        return {QStringLiteral("label")};
+    }
+
+    if (mpConsole->mSubConsoleMap.contains(name)) {
```

Comment:
There is a method I put in place to help with identifying the type of a `TConsole`:
`(TConsole::ConsoleType) TConsole::getType(const QString& name)`
why not use that?

Reply:
We still have a lot of code that hasn't been upgraded to use it so I missed it!